lib/ckb/wallet.rb

```diff
@@ -128,7 +128,7 @@ def generate_tx(target_address, capacity, data = "0x", key: nil, fee: 0, use_dep
     # @param data [String] "0x..."
     # @param key [CKB::Key | String] Key or private key hex string
     # @param fee [Integer] transaction fee, in shannon
-    def send_capacity(target_address, capacity, data = "0x", key: nil, fee: 0, outputs_validator: "default", from_block_number: 0)
+    def send_capacity(target_address, capacity, data = "0x", key: nil, fee: 0, outputs_validator: "passthrough", from_block_number: 0)
```

Comment:
I believe it is safer to set the default value of outputs_validator to 'default'

Reply:
CKB provides two built-in validators, well_known_scripts_only and passthrough. When no validator is specified, well_known_scripts_only is used.

you means set `well_known_scripts_only` to be default?
now `outputs_validator: "default"` will report an error.